bears/general/TextLintBear.py

```diff
@@ -0,0 +1,193 @@
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar', 'Spelling'}
+    SEE_MORE = 'https://github.com/textlint/textlint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        keyword_todo: bool=None,
+                        no_start_duplicated_conjunction: bool=True,
+                        no_empty_section: bool=True,
+                        check_date_weekday_mismatch: bool=True,
+                        check_grammar: bool=True,
+                        max_lines_per_file: int=300,
+                        max_comma_per_sentence: int=4,
+                        check_no_good_words: typed_list(str)=[],
+                        period_in_list_item: bool=True,
+                        minimum_acronym_length: int=3,
+                        maximum_acronym_length: int=5,
+                        ignore_acronyms: typed_list(str)=[],
+                        check_rousseau: bool=True,
+                        check_offensive_expressions: bool=True,
+                        check_common_misspellings: bool=True,
+                        write_good: bool=True,
+                        check_relative_links: bool=False,
+                        base_uri: str='null',
+                        ignore_links: typed_list(str)=[],
+                        textlint_config: str='',
+                        ):
+        """
+        :param keyword_todo:
+            This rule checks ``- [ ]`` (task lists).
+        :param no_start_duplicated_conjunction:
+            This rule checks whether your sentence starts with a duplicated
+            conjunction.
+        :param no_empty_section:
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header B`` below:
+            ```
+            # Header A
+
+            # Header B
+
+            Text.
+            ```
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between a date and the corresponding
+            weekday.
+        :param check_grammar:
+            This rule checks your English grammar with Ginger Proofreading.
+        :param max_lines_per_file:
+            Number of lines allowed per file.
+        :param max_comma_per_sentence:
+            Number of commas allowed per sentence.
+        :param check_no_good_words:
+            Set of NG (No Good) words to check for.
+        :param period_in_list_item:
+            This rule checks whether there is a period in a list item.
+        :param minimum_acronym_length:
+            Minimum length for the unexpanded acronym.
+        :param maximum_acronym_length:
+            Maximum length for the unexpanded acronym.
+        :param ignore_acronyms:
+            A list that contains the acronyms to ignore.
+        :param check_rousseau:
+            This rule checks English sentence using rousseau, which is a
```

Comment:
is "roussou" written lower-case? If it's a usual name, we should start it with an upper-case letter :+1: